phoenix-core-client/src/main/java/org/apache/phoenix/execute/MutationState.java

```diff
@@ -1489,8 +1502,21 @@ public List<Mutation> getMutationList() {
                             }, iwe, connection, connection.getQueryServices().getProps());
                             shouldRetryIndexedMutation = false;
                         } else {
-                            hTable.batch(mutationBatch, null);
+                            hTable.batch(mutationBatch, resultObjects);
+                        }
+
+                        if (resultObjects != null) {
+                            Result result = (Result) resultObjects[0];
+                            if (result != null && !result.isEmpty()) {
+                                Cell cell = result.rawCells()[0];
+                                numUpdatedRowsForAutoCommit = PInteger.INSTANCE.getCodec()
+                                        .decodeInt(cell.getValueArray(), cell.getValueOffset(),
+                                                SortOrder.getDefault());
```

Comment:
Here also, we need to use `_UpsertCF:_UpsertStatusCQ` to get the Cell

Reply:
Done